packages/node_modules/@node-red/util/lib/util.js

```diff
@@ -413,31 +413,48 @@ function setObjectProperty(msg,prop,value,createMissing) {
     }
 }
 
+/**
+ * Get value of environment variable.
+ * @param {Node} node - accessing node
+ * @param {String} name - name of variable
+ * @return {String} value of env var
+ */
+function getenv(node, name) {
```

Comment:
- This should call `node._flow.getSetting(name)`.
 - Please use camelCase when adding Functions.

Reply:
Changed to use `getSetting`.
Name of the function also changed to `getSetting` from `getenv`.